docs/site/Server.md

```diff
@@ -21,13 +21,12 @@ listening on different ports and working with different protocols.
 
 LoopBack 4 offers the
 [`@loopback/rest`](https://github.com/strongloop/loopback-next/tree/master/packages/rest)
-package out of the box, which provides HTTP/HTTPS based server called
+package out of the box, which provides an HTTP/HTTPS-based server called
```

Comment:
`an` => `a`

Reply:
"A HTTP" vs "An HTTP" It can go either way, depending upon your pronunciation. I say "aitch" for H; I guess there is an alternate pronunciation that says "haitch". This is from the Chicago Manual of Style "If ... the acronym is treated as a series of letters, the choice of the article depends on the pronunciation of the first letter."

I prefer "An" but really could change it to "a" if you want me to!